entity-service/src/main/kotlin/com/egm/stellio/entity/web/EntityOperationHandler.kt

```diff
@@ -27,6 +24,7 @@ import java.net.URI
 class EntityOperationHandler(
     private val entityOperationService: EntityOperationService,
     private val authorizationService: AuthorizationService,
+    private val entityHandler: EntityHandler,
```

Comment:
I don't like this, but i can't find another way to avoid duplicating publishAppendEntityAttributesEvents function.

Reply:
Can't you move publishAppendEntityAttributesEvents  to another class  ? I know that you have to take care of dependency cycle, but it will improve reusability

Reply:
Indeed. Why not in EntityEventService?

Reply:
Yes for EntityEventService